roles/validations/tasks/main.yml

```diff
@@ -56,5 +68,34 @@
       failed_when: not validation_exists.stat.exists
 
     - name: Run validations
-      ansible.builtin.include_tasks: "{{ item }}"
+      ansible.builtin.include_tasks:
+        file: xmlcreate.yml
       loop: "{{ cifmw_validations_list }}"
+
+- name: Get validations end time from system
+  ansible.builtin.command:
+    cmd: date +%Y-%m-%d-%H:%M:%S.%3N
+  register: validations_end_time
+
+- name: Calculate total validations run time
+  ansible.builtin.set_fact:
+    validations_run_time: "{{((validations_end_time.stdout | to_datetime('%Y-%m-%d-%H:%M:%S.%f')) - (validations_start_time.stdout | to_datetime('%Y-%m-%d-%H:%M:%S.%f'))).total_seconds()}}"
+
+- name: Write first line of xml results file
+  ansible.builtin.shell: |
+    mkdir -p "{{ cifmw_validations_xml_status_file_dir }}"
```

Comment:
` mkdir -p "{{ cifmw_validations_xml_status_file_dir }}"` Why using a shell to do something Ansible is designed for?

Reply:
Used ansible.builtin.file instead